### DISCUSSION TITLE
Fix Send audio/WhatsApp web

### DISCUSSION THREAD

**Discussion by m3m2020** [AndreyPavlenko/Fermata#329] [General]
Hello, thanks for your work.  There is a problem, in WhatsAppweb, the back icon is always above the microphone icon, so you cannot press it to send audio, would it be possible to move it a little higher or choose the specific place to be able to send an audio easily?  (I have tried it in three different cars) thanks

**Comment by AndreyPavlenko** [comment]
You can move the back button - long press on it and drag.

**Reply by m3m2020** [reply]
How did I not try it??
Thank you, I take my hat off to you :)
